views/layout.ejs

```diff
@@ -23,7 +23,6 @@
     <link rel="stylesheet" href="/assets/dist/octofarm.css"/>
     <title><%= octoFarmPageTitle %>: <%= page %></title>
     <!-- Avoid trouble -->
-    <script src="/assets/dist/client-error.modal.min.js"></script>
```

Comment:
This a mistake perhaps?

Reply:
No, the page isn't ready to grab the elements it uses so I have to grab them everytime requested or can't trigger the jQuery bootstrap modals.